src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -175,7 +177,7 @@ public void deleteInterviewForCandidate(Candidate target) {
 
     @Override
     public void deleteInterview(Interview interviewToDelete) {
-        interviewSchedule.removeInterview(interviewToDelete);
+        Candidate candidate = interviewSchedule.removeInterview(interviewToDelete);
```

Comment:
Hi @tiewweijian! May I ask what is the `candidate` variable here used for? I don't see it being used in the code, unless I am missing something! (also the method is returning `void`, but more importantly I don't see `candidate` variable being accessed`.

Reply:
oh my bad !! I did some changes just now and forgot to revert ! Pushing up a new commit to change this !